Userland/Libraries/LibGUI/VimEditingEngine.cpp

```diff
@@ -971,13 +971,22 @@ bool VimEditingEngine::on_key_in_normal_mode(const KeyEvent& event)
                 move_one_up(event);
                 switch_to_insert_mode();
                 return true;
-            // FIXME: Integrate these into vim motions too.
-            case (KeyCode::Key_LeftBrace):
-                move_to_previous_empty_lines_block();
+            case (KeyCode::Key_LeftBrace): {
+                auto amount = m_motion.amount() ? m_motion.amount() : 1;
```

Comment:
Consider making this behavior more obvious:
```suggestion
                auto amount = m_motion.amount() > 0 ? m_motion.amount() : 1;
```

Reply:
Agreed that makes sense; updated